stdlib/public/core/Integers.swift

```diff
@@ -3010,9 +3010,7 @@ extension FixedWidthInteger {
     let minBitWidth = source.significandWidth
     let isExact = (minBitWidth <= exponent)
     let bitPattern = source.significandBitPattern
-    // `RawSignificand.bitWidth` is not available if `RawSignificand` does not
-    // conform to `FixedWidthInteger`; we can compute this value as follows if
-    // `source` is finite:
+    // Determine the number of meaningful bits in the significand bit pattern.
```

Comment:
Good catch. For whatever reason, I was actually intending to compute the bit width of `RawSignificand`, but that has since been reworked.

The "number of meaningful bits" would probably be interpreted by most to mean `significandWidth` (i.e., `minBitWidth`); let's use existing terminology here:

```suggestion
    // Determine the actual number of fractional significand bits.
```

You might then ask, why not use `Source.significandBitCount`? The answer (as I recall) is that it wouldn't be correct for a future extensible-width floating-point type, where that API is supposed to return `Int.max` instead of the actual significand bits, since the latter would vary from value to value. For the same reason, `RawSignificand` does not conform to `FixedWidthInteger`, and hence the comment here in the first place. This would be worth noting:

```
// `Source.significandBitCount` would not reflect the actual number of
// fractional significand bits if `Source` is not a fixed-width floating-point
// type; we can compute this value as follows if `source` is finite:
```

Reply:
That makes sense! Added the expanded clarification.